pkg/command/install/install.go

```diff
@@ -157,3 +124,162 @@ func getOperatorURL(version string) (string, error) {
 	}
 	return URL, nil
 }
+
+func getOverlayYamlContent(installFlags installCmdFlags, rootPath string) string {
+	path := ""
+	if strings.EqualFold(installFlags.Component, "serving") {
```

Comment:
And this will work with the operator’s binary file only?

Reply:
It does not need any binary to work.
If users use kn-operator-plugin, this plugin wraps-up everything underneath, like fetching the operator, installing the components, configuring the components, etc.